gh-105052:update timeit function's description (#105060)
python/cpython: Doc/library/timeit.rst

      Time *number* executions of the main statement.  This executes the setup
      statement once, and then returns the time it takes to execute the main

>       statement a number of times, measured in seconds as a float by default timer.

Reviewer: ```suggestion
      statement a number of times.  The default timer returns seconds as a float.
```

Or see other suggestions on issue.
